### ISSUE TITLE
VO is not announcing about "Error Popup" obtained while changing working platfrom instead it sets focus on "OK" button reads about it.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#271]
### Steps to Reproduce

1. Launch Xamarin Workbook application-> Select Mac platform and create a new workbook.
Turn On the VoiceOver.
2. Navigate to "Working Platform (Environment)" dropdown present in the tool bar and change the platform (From Mac full to Console (.NET Core) ) which throws "Error Popup" i.e. (Error trying to inspect remote view , Index was outside the bounds of the array).
3. Observe the VO behaviour

### Expected Behavior
VO Should announce about Error description. The VoiceOver should read the error alert also, without hitting on any keys.

### Actual Behavior
Once the popup appears, VO sets focus on OK button and reads as OK button.
Note:  Although if user hits on Control+option+up arrow, the VoiceOver is announcing the error alert.

### User Impact:
User won't be able to know that error popup is popped up until its automatically being read by the  VoiceOver. 
Note: The alert text is having two lines only, so it would be easier for the VoiceOver user if the error alert text is read by the VoiceOver.

### Environment
MAC: macOS High Sierra [10.13.4]
Workbook - 1.4 (master/b3f92f9) 

### Log File
[VO9_ErrorPopup_Console.zip](https://github.com/Microsoft/workbooks/files/1990903/VO9_ErrorPopup_Console.zip)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS3.3.1
Xamarin-Workbook-MAC
Wipro

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1823
